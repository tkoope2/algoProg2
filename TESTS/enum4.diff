7a8,17
> @: v1 v2 v3 v4 v5 v6
> @: v1 v2 v3 v4 v6
> @: v1 v2 v3 v5 v6
> @: v1 v2 v4 v5 v6
> @: v1 v2 v4 v6
> @: v1 v2 v6
> @: v1 v3 v4 v5 v6
> @: v1 v3 v4 v6
> @: v1 v3 v5 v6
> @: v1 v5 v6
