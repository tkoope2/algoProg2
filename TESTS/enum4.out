@1++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
@2
@3 COMMAND-LINE:
@4
@5    ./epaths chain20.g v6 
@6
@7++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
